Tools/clinic/clinic.py

```diff
@@ -887,7 +895,7 @@ def parser_body(prototype, *fields, declarations=''):
                         """, indent=4)]
                 else:
                     parser_code = [normalize_snippet("""
-                        if (!PyArg_ParseTuple(args, "{format_units}:{name}",
+                        if (!_PyArg_ParseTupleAndKeywordsFast(args, kwargs, &_parser,
```

Comment:
Why?

Reply:
Use the limited C API when using the use_limited_api flag, otherwise it'll use the unlimited C API. 

Sorry for my delay.